### DISCUSSION TITLE
make it easy to add an RSS/ATOM feed to a static site

### DISCUSSION THREAD

**Discussion by haroldcarr** [srid/emanote#394] [Ideas]
I really like reading people's blogs/web sites.   But not if I have to check them every day to see if there is new content.   That's what RSS/Atom is for - so I see new content in my feed reader.

Does ema and emanote support easily adding RSS/Atom feeds?
If not, it would be great to add that support.

BTW: my feed reader is https://bazqux.com/
bazqux is written in Haskell for the backend and Ur/Web for the front end http://www.impredicative.com/ur/

**Comment by srid** [comment]
Note to self: adapt https://neuron.zettel.page/feed in some fashion to Emanote queries.
